src/components/dropdown_styles.css

```diff
@@ -0,0 +1,148 @@
+/* Dropdown Button */
+.dropbtn {
+display: fixed;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+padding: 0px;
+gap: 8px;
+width: 98px;
+height: 40px;
+background: #006D7D;
+border-radius: 8px;
+flex: none;
+order: 0;
+align-self: flex-start;
+flex-grow: 0;
+margin-left: 100px;
+margin-top: 100px;
+border: none;
+}
+
+.C1 {
+   color: white;
+   font-style: normal;
+   font-size: 14px;
+   margin-left: 7px;
+   position: relative;
+   bottom: 2px;
+   
+}
+
+
+#Change{
+   color: white;
+   position: relative;
+   top: 2px;
+}
+
+
+.items {
+width: 116px;
+height: 24px;
+font-family: "Roboto";
+font-style: normal;
+font-weight: 400;
+font-size: 16px;
+line-height: 24px;
+display: flex;
+align-items: center;
+letter-spacing: 0.5px;
+color: #1C1B1F;
+flex: none;
+order: 1;
+align-self: stretch;
+flex-grow: 0;
+}
+
+ul {
+   list-style-type: none;
+   padding: 0;
+   padding-left: 16px;
+   padding-bottom: 18px;
+}
+
+li {
+   margin: 8px 8px 0px 0;
+   margin-bottom: 26px;
+   margin-top: -9px;
+}
+
+input[type=checkbox] {
+   position: absolute;
+   cursor: pointer;
+   margin-left: 100px;
+   margin-bottom: 10px;
+   margin-left: 135px;
+}
+
+input[type=checkbox]:before {
+   box-sizing: border-box;
+   border: 2px solid #006D7D;
+   border-radius: 2px;
+   content: "";
+   display: block;
+   position: absolute;
+   width: 20px;
+   height: 20px;
+   top: 0;
+   left: 0;
+   background-color:white;
+   border-width: 2px;
+}
+
+input[type=checkbox]:checked:before {
+   content: "";
+   display: block;
+   position: absolute;
+   width: 20px;
+   height: 20px;
+   top: 0;
+   left: 0;
+   background-color:#006D7D;
+}
+
+input[type=checkbox]:checked:after {
+   content: "";
+   display: block;
+   width: 5px;
+   height: 10px;
+   border: solid white;
+   border-width: 0 2px 2px 0;
+   -webkit-transform: rotate(45deg);
+   -ms-transform: rotate(45deg);
+   transform: rotate(45deg);
+   position: absolute;
+   top: 2px;
+   left: 6px;
+}
+
+ .dropdown {
+   position:absolute;
+   display: flexbox;
+   width: 100px;
+   left: 100px;
+ }
+
+
+ /* Dropdown Content (Hidden by Default) */
+ .dropdown-content {
```

Comment:
for the dropdown content, maybe we should have a border outline so its visible. The background is planned to be white so it would be hard to see.

Reply:
after changing the margin for the button, the dropdown doesn't drop right below the button anymore. We need to somehow "attach" the menu so that it always appears below the button, no matter where we end up putting the button